ray-operator/controllers/ray/raycluster_controller_unit_test.go

```diff
@@ -2025,10 +2041,10 @@ func Test_ShouldDeletePod(t *testing.T) {
 			// The restart policy is `Always` and the Pod is in a terminate state.
 			// The expected behavior is that the controller will not delete the Pod because
 			// the restart policy is `Always`.
```

Comment:
The comments here should be updated. The controller will delete the pod regardless the restart policy.